README.md

```diff
@@ -1,2 +1,29 @@
-# WebApp
-@CMNatic @Ju3tJames
+# Web App
+# @CMNatic @Ju3tJames
+
+[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)
+
+Development web app / manager for Docker
+
+  - Stop, start and export Images and Containers
+  - Docker logs
+  - Launching containers via DockerFile with Environment variables
+
+# TO DO:
+  - MAKE THE FUCKING THING
+  - that
+  - MYSQL Backend, see if we can use e.g. MongoDB
```

Comment:
Should we use MongoDB? Shipyard use Rethink, maybe we should think about using a  lightweight database